src/compas_model/model/elementtree.py

```diff
@@ -44,7 +44,6 @@ def __init__(self, model, name=None):
         super(ElementTree, self).__init__(name=name)
         root = GroupNode(name="root")
         self.add(root)
-        self.model = model
```

Comment:
@tomvanmele @petrasvestartas not sure if this is needed. It was causing an issue with deep copying/serialization.
The reason was `Model.__data__` was called twice. Once by `Data.copy()` and once again by `deepcopy` when invoking it on the `ElementTree.__dict__` which had an instance of `Model. Because `__data__`  replaces element references to elements with their guids, the second run would fail trying to interact with `Element` instances.

I removed it as I saw it's not used anyways, but if you're intending to restore and make use in the future then just keep that in mind.